devel/contents.tex

```diff
@@ -48,10 +48,16 @@ \section{Examination}%
 The report will be graded according to the following grading criteria:
 \begin{description}
   \item[Grade E] You fulfil all the \acp{ILO} at the minimum level.
+
+  \item[Grade C] You fulfil all the \acp{ILO},
+    you evaluations and designs are \emph{good} with \emph{some base} in theory 
+    and, where applicable, the research literature and otherwise show 
+    \emph{some} insights in how to use the building-blocks from the literature.
+
   \item[Grade A] You fulfil all the \acp{ILO},
-    your evaluations and designs are extensive and well-founded in theory and, 
-    where applicable, the research literature and otherwise show deep insights 
-    in how to use the building-blocks from the literature.
+    your evaluations and designs are \emph{extensive} and \emph{well-founded} 
```

Comment:
Where in the ILO is it mentioned a quantitative goal? What is extensive? If their work is based on just one article but they identify mistakes and improves it? Is this not enough for an A?

Reply:
Yes, and that precisely illustrates the point of why I don't want to make the criteria more concrete, because then I'd need an enormous list of cases OR-ed together.

It's the evaluations that should be extensive (not the designs, that's a result of bad phrasing). I'd say that identifying a mistake in a paper and fixing it would probably require an extensive evaluation, in either case the fix would also be well-founded: "this is wrong, because ..., we should fix it like this ..." (extensive and well-founded) as opposed to "this looks weird, maybe we should do something different".

Reply:
> > Some comments, tried to give some suggestions on improvements.
> > The aims in the description is not aligned to the ILOs in the course syllabus however. Needs to be corrected.
> 
> Do you mean the syllabus or what's stated in the study guide? It's what's in the study guide that should make it into the syllabus.



> > Some comments, tried to give some suggestions on improvements.
> > The aims in the description is not aligned to the ILOs in the course syllabus however. Needs to be corrected.
> 
> Do you mean the syllabus or what's stated in the study guide? It's what's in the study guide that should make it into the syllabus.

Syllabus, the one in the study guide is correct. Just a reminder to update the ILO in ATLAS.